lib/foreman_salt/plugin.rb

```diff
@@ -144,5 +144,14 @@
   parameter_filter Host::Managed, :salt_proxy_id, :salt_proxy_name,
     :salt_environment_id, :salt_environment_name, :salt_modules => [],
     :salt_module_ids => []
+
+  if ForemanSalt.with_remote_execution? && Gem::Version.new(ForemanRemoteExecution::VERSION) >= Gem::Version.new('1.2.3')
```

Comment:
Why specifically `REX >= 1.2.3`?

Reply:
adapted from foreman_openscap plugin and similar to foreman_chef.